subsys/bluetooth/host/iso.c

```diff
@@ -1850,13 +1850,13 @@ static int hci_le_big_create_sync(const struct bt_le_per_adv_sync *sync, struct
 	req->sync_timeout = sys_cpu_to_le16(param->sync_timeout);
 	req->num_bis = big->num_bis;
 	/* Transform from bitfield to array */
-	for (int i = 0; i < BT_ISO_MAX_GROUP_ISO_COUNT; i++) {
+	for (int i = 1; i < BT_ISO_MAX_GROUP_ISO_COUNT; i++) {
```

Comment:
shouldn't this be changed to 
```suggestion
	for (int i = 1; i <= BT_ISO_MAX_GROUP_ISO_COUNT; i++) {
```